Tests/MessagingPush/PushHandling/AutomaticPushClickedIntegrationTest.swift

```diff
@@ -4,7 +4,9 @@ import Foundation
 import SharedTests
 import XCTest
 
-class AutomaticPushClickHandlingIntegrationTest: IntegrationTest {
+// Integration tests that simulate a push being clicked on a device and
+// how our SDK handles that event.
+class AutomaticPushClickedIntegrationTest: IntegrationTest {
```

Comment:
Change class name because lint said the class name was too long (> 40 characters).